site/src/examples/navigation-item/VerticalNestedGroup.tsx

```diff
@@ -23,7 +23,15 @@ export const VerticalNestedGroup = (): ReactElement => {
 
   return (
     <nav>
-      <ul style={{ width: 250, listStyle: "none", paddingLeft: 0 }}>
+      <StackLayout
```

Comment:
StackLayout isn't really giving us a lot of benefits here over just:

display: flex;
flex-drection: column

Reply:
gotcha - but the pattern also says [https://www.saltdesignsystem.com/salt/patterns/vertical-navigation#layout ](Vertical navigation uses StackLayout to display navigation items in a column formation.)